### DISCUSSION TITLE
Request chaining

### DISCUSSION THREAD

**Discussion by michbsd** [darrenburns/posting#132] [Ideas]
Would be awesome if you would consider supporting request chaining.

So request B can dynamically use the value from request A's response header

**Comment by darrenburns** [comment]
You could do this with scripting at the moment. Run request A, extract the header and set it as a variable, then run request B and make use of that variable. There's some examples in the scripting section of the docs.

It's not clear to me if chaining is something different.

**Comment by michbsd** [comment]
Yes, that it is.
I saw that there was some scripting around this, thanks - will have a look at it.
Though, I would suggest to have this as a native feature in the tool at some point 

thanks again for bringing this tool to us
